xml/System.Runtime.Intrinsics.X86/Avx10v1+X64.xml

```diff
@@ -40,7 +40,7 @@
       <Docs>
         <param name="upper">To be added.</param>
         <param name="value">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>__m128d _mm_cvtsi64_sd (__m128d a, __int64 b) VCVTUSI2SD xmm1, xmm2, r/m64 This intrinsic is only available on 64-bit processes</summary>
```

Comment:
Should "This intrinsic is only available on 64-bit processes" be in the remarks instead?

Reply:
I think there's a lot of missing newlines or `<para>` separators in these as well

Reply:
@tannergooding I don't see the comments in triple slash split by `<para>`.

Reply:
They're split by newline in the triple slash, which I don't think has been an issue with the other intrinsic docs; unless we manually fixed those all up in the docs?

Reply:
I think there's a bug in the tool. I was expecting it to split new lines using `<para>`. I fixed it manually. Can you please take another look?